tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,29 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """Collections page."""
+
+    _item_locator = (By.CSS_SELECTOR, '.item')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: len(self.collections) > 0 and
+                        self.collections[0].name is not None)
```

Comment:
`self.collections[0].name` will never be `None`, but it could be an empty string if the element is not displayed. You could just check that it's truthy.